docs/content/config/environment.md

```diff
@@ -715,3 +715,5 @@ you to replace both instead of just the envelope sender.
 
 - **empty** => no default
 - password for default relay user
+
+[docs-faq-onedir]: ../faq.md#what-about-mail-state-folder
```

Comment:
```suggestion
[docs-faq-onedir]: ../faq/#what-is-the-mail-state-folder-for

Reply:
IIRC, it expects to be given the `.md` extension, it'll convert to `faq/` during build. It'd be good if I ever find time to implement the lint test for `#anchor` references in the link URI :sweat_smile: 

```suggestion
[docs-faq-onedir]: ../faq.md#what-is-the-mail-state-folder-for
```

Thanks for spotting this one! :heart: